std/typecons.d

```diff
@@ -2522,6 +2527,12 @@ Returns:
         return _value;
     }
 
+    /// ditto
+    @property get(T)(T datum) inout @safe pure nothrow
```

Comment:
You need to mark datum as `inout` as well, or else the return will always be `const` (because of the inferred type of the `?:` expression).